hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/NameNode.java

```diff
@@ -949,13 +947,13 @@ protected void startMetricsLogger(Configuration conf) {
       return;
     }
 
-    MetricsLoggerTask.makeMetricsLoggerAsync(MetricsLog);
+    MetricsLoggerTask.makeMetricsLoggerAsync(METRICS_LOG);
```

Comment:
again, a version of this method which took a log name would handle this internally.

Reply:
makes sense